cloudinary-core/src/main/java/com/cloudinary/Uploader.java

```diff
@@ -265,9 +265,19 @@ public Map generate_sprite(String tag, Map options) throws IOException {
     }
 
     public Map generateSprite(String tag, Map options) throws IOException {
+        options.put("tag", tag);
+        return generateSprite(options);
+    }
+
+    public Map generateSprite(String[] urls, Map options) throws IOException {
```

Comment:
What happens if I send both `urls` and `tag`?

Reply:
See my comment below about validations. Regardless, when calling one of the two public method you pass a parameter directly (not through the map). In order to send both you'd have to manually add the other one to the map - going down this path it's pretty easy to break all the requests.